org.nickvision.tagger.json

```diff
@@ -0,0 +1,104 @@
+{
+    "app-id" : "org.nickvision.tagger",
```

Comment:
the domain `nickvision.org` doesn't exist. This app-id doesn't meet the requirements.